data_bootstrap/management/commands/bootstrap.py

```diff
@@ -0,0 +1,20 @@
+import os
+import glob
+
+from django.core.management import BaseCommand
+
+from sil_data_bootstrap.bootstrap import process_json_file
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        for suggestion in args:
+            # if it's just a simple json file
+            if os.path.exists(suggestion) and os.path.isfile(suggestion):
+                process_json_file(suggestion)
+            else:
+                # check if it's a glob
+                for filename in glob.glob(suggestion):
+                    process_json_file(filename)
+        print "Done loading"
```

Comment:
Do not use `print` in management commands. I'll let the following excerpt from the Django docs speak for itself:

```
Note

When you are using management commands and wish to provide console output, you should write to self.stdout and self.stderr, instead of printing to stdout and stderr directly. By using these proxies, it becomes much easier to test your custom command. Note also that you don’t need to end messages with a newline character, it will be added automatically, unless you specify the ending parameter:

self.stdout.write("Unterminated line", ending='')
```

PS: Backport that change to the source of this library.